Makefile

```diff
@@ -100,13 +100,13 @@ start-docs: ## Start the local mkdocs based development environment.
 
 .PHONY: clean-docs
 clean-docs: ## Remove all local mkdocs Docker images (cleanup).
-	docker container prune --force --filter "label=project=virtlet_documentation"
+	docker container prune --force --filter "label=project=libvirt-provider_documentation"
 
 ##@ Build
 
 .PHONY: build
 build: generate fmt add-license lint ## Build the binary
-	GOOS=$(TARGET_OS) GOARCH=$(TARGET_ARCH) go build -o bin/virtlet ./main.go
+	GOOS=$(TARGET_OS) GOARCH=$(TARGET_ARCH) go build -o $(LIBVERTPROVIDERBIN) ./main.go
```

Comment:
```suggestion
	GOOS=$(TARGET_OS) GOARCH=$(TARGET_ARCH) go build -o $(LIBVIRTPROVIDERBIN) ./main.go
```

Reply:
Trupti did you test make build? because path for main.go doesn't look correct

Reply:
Good catch. The main.go file is actually located under `provider/cmd`.

Reply:
corrected the path of main file. make build is running fine.